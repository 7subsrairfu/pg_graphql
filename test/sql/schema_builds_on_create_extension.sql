drop extension pg_graphql;
create extension pg_graphql;

select jsonb_pretty(
    graphql.resolve($$
    { heartbeat }
    $$)
);
