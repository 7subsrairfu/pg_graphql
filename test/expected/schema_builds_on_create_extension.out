drop extension pg_graphql;
create extension pg_graphql;
select jsonb_pretty(
    graphql.resolve($$
    { heartbeat }
    $$)
);
                   jsonb_pretty                    
---------------------------------------------------
 {                                                +
     "data": {                                    +
         "heartbeat": "2022-04-26T16:11:30.088137"+
     }                                            +
 }
(1 row)

